code/script.js

```diff
@@ -1,325 +1,539 @@
-    hair: 'grey',
+    img: 'image/character-21.png',
+    hair: 'hidden',
     eyes: 'brown',
-    accessories: [],
-    other: []
+    accessories: ["glasses"],
+    job: []
   },
   {
     name: 'Josh',
-    img: 'images/josh.svg',
-    hair: 'yellow',
+    img: 'image/character-22.png',
+    hair: 'black',
     eyes: 'green',
-    accessories: [],
-    other: []
+    accessories: ["hairband"],
+    job: []
   },
   {
     name: 'Jude',
-    img: 'images/jude.svg',
-    hair: 'black',
+    img: 'image/character-23.png',
+    hair: 'red',
     eyes: 'green',
     accessories: [],
-    other: []
+    job: ["singer"]
   },
   {
     name: 'Julie',
-    img: 'images/julie.svg',
-    hair: 'black',
+    img: 'image/character-24.png',
+    hair: 'hidden',
     eyes: 'brown',
-    accessories: ['glasses', 'hat'],
-    other: []
+    accessories: [],
+    job: ["gamer"]
   },
 ]
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret;  // Secret person
+let currentQuestion; 
+let charactersInPlay; //charactersInPlay = CHARACTERS
+let playerName;  // To store user's name
+
+
+//1. Name Input Fotm
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  playerName = nameInput.value;
+  greeting.textContent = `Hi ${playerName}, let's get started`
+  magicSound.play();
+  music.play()
+  // To hide the name input form
+  form.style.display = "none";
+
+  // To start the game
+  setTimeout(start,1000)
+})
+
+
+
+
+// 2. Create the timer
+let totalSecond =0;
+let numberOfGuess = 0;
+
+// Create a function to store and display the time
+function countTime() {
+
+  // Get the total time in second
+  totalSecond++;
+
+  // Divide the total amount and grab the remainder
+  // We don't want to show the whole time in second, only the one less than 60
+  const sec = totalSecond % 60;
 
-// Draw the game board
+  // Divide the total amount to 60 to get the minute. Use Math.floor to ground the number
+  const min = Math.floor((totalSecond/60));
+
+  // Set up how to display second
+  if (sec < 10) {
+    second.textContent = `0${sec}`
+  } else  {
+  second.textContent = sec
+  }
+  
+  // Set up how to display minute
+  if (min < 10) {
+    minute.textContent = `0${min} : `
+  } else {
+    minute.textContent = `${min} : `
+  }
+}
+
+// Invoke the function after every 1 second
+
+setInterval(countTime,1000)
+
+
+// 3. Draw the game board
 const generateBoard = () => {
   board.innerHTML = ''
   charactersInPlay.forEach((person) => {
     board.innerHTML += `
-      <div class="card">
+      <div class="card" onmouseover="playSound()">
         <p>${person.name}</p>
         <img src=${person.img} alt=${person.name}>
         <div class="guess">
           <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <button class="filled-button small guess-btn" onclick="guess('${person.name}')" >Guess</button>
```

Comment:
`class="filled-button small guess-btn"`
Kind of long, might be easier to just give this button a new class?